### PR TITLE
proposing change

### PR DESCRIPTION
**Pull request by 4c74356b41**
Actually, after reading through this article I feel like it looks more like a blog post, rather than a piece of documentation ;) And I know you did a lot of great blog posts (just like this one)
Don't take my words as a animadversion, you did a really great job, but I don't think I've ever seen Azure documentation article that is so personal (in a sense that you use "my", "I" all the time). I will try to fix that, and you are free to reject my ideas.

ps. Great article, mate!

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/site-recovery/site-recovery-automate-mobility-service-install.md [hunk 1]
```diff
@@ -477,7 +477,7 @@ If you want to instantiate your own DSC pull server within your corporate networ
 
 ## Optional: Deploy DSC Configuration using Azure Resource Manager Template
 
-In this article, we have so far been focusing on how you can create your own DSC configuration to automatically deploy the Mobility Service and the Azure VM Agent – and ensure they are running on the machines you would like to protect. As a bonus, we have also an Azure Resource Manager##  ## template that will deploy this DSC configuration to a new or existing Azure Automation account, creating automation assets that will contain the variables for your environment through input parameters in the template.
+In this article, the focus has been on how you can create your own DSC configuration to automatically deploy the Mobility Service and the Azure VM Agent – and ensure they are running on the machines you would like to protect. As a bonus, we have also an **Azure Resource Manager** template that will deploy this DSC configuration to a new or existing Azure Automation account, creating automation assets that will contain the variables for your environment through input parameters in the template.
```
Comment (4c74356b41) [inline articles/site-recovery/site-recovery-automate-mobility-service-install.md]: I undid ## ## to ** ** change in the next commit and did a different PR to attempt to fix it (as i'm not sure what the original intention was).

**Comment by PRmerger** [comment]
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @krnese, has been notified to review your proposed change.

**Comment by krnese** [comment]
#sign-off

**Comment by 4c74356b41** [comment]
Wow, I'm actually pretty excited about this merge, thanks ;)

**Pull request merged**